proj-xcode/PowerAuth2/PowerAuthSDK.m

```diff
@@ -1766,35 +1762,28 @@ @implementation PowerAuthSDK (UserInfo)
 - (PowerAuthUserInfo*) lastFetchedUserInfo
 {
     [_lock lock];
```

Comment:
I think the lock - unlock is no longer required, because you moved storage to C++ session, which is already thread safe.